tests/testthat/test-ppc-distributions.R

```diff
@@ -177,6 +193,38 @@ test_that("ppc_boxplot renders correctly", {
   vdiffr::expect_doppelganger("ppd_boxplot (alpha, size)", p_custom)
 })
 
+test_that("ppc_qdotplot renders correctly", {
+  testthat::skip_on_cran()
+  testthat::skip_if_not_installed("vdiffr")
+  testthat::skip_if_not_installed("ggdist")
+  skip_on_r_oldrel()
+
+  p_base <- ppc_qdotplot(vdiff_y, vdiff_yrep[1:8, ])
+  vdiffr::expect_doppelganger("ppc_qdotplot (default)", p_base)
+
+  p_binwidth <- ppc_qdotplot(vdiff_y, vdiff_yrep[1:8, ], binwidth = 3)
+  vdiffr::expect_doppelganger("ppc_qdotplot (binwidth)", p_binwidth)
+
+  p_quantile <- ppc_qdotplot(vdiff_y, vdiff_yrep[1:8, ], quantiles = 50)
+  vdiffr::expect_doppelganger("ppc_qdotplot (quantile)", p_quantile)
+
+  p_quantile_binwidth <- ppc_qdotplot(vdiff_y, vdiff_yrep[1:8, ], binwidth = 3, quantiles = 50)
+  vdiffr::expect_doppelganger("ppc_qdotplot (quantile-binwidth)", p_quantile_binwidth)
+
+  # ppd versions
+  p_base <- ppd_qdotplot(vdiff_yrep[1:8, ])
+  vdiffr::expect_doppelganger("ppd_qdotplot (default)", p_base)
+
+  p_binwidth <- ppd_qdotplot(vdiff_yrep[1:8, ], binwidth = 3)
+  vdiffr::expect_doppelganger("ppd_qdotplot (binwidth)", p_binwidth)
+
+  p_quantile <- ppd_qdotplot(vdiff_yrep[1:8, ], quantiles = 50)
+  vdiffr::expect_doppelganger("ppd_qdotplot (quantile)", p_quantile)
+
+  p_quantile_binwidth <- ppd_qdotplot(vdiff_yrep[1:8, ], binwidth = 3, quantiles = 50)
+  vdiffr::expect_doppelganger("ppd_qdotplot (quantile-binwidth)", p_quantile_binwidth)
+})
+
```

Comment:
Same thing as above about warning messages. Aside from checking that they throw a warning when we expect them too, the logs test output also gets very messy when we have a lot of warnings. Wrapping them is expect_warning() will get rid of that.